src/administration/vouchers.md

```diff
@@ -0,0 +1,13 @@
+# Vouchers 
+
+## Applying a voucher to your project
+
+If you receive a [Platform.sh](https://platform.sh/) voucher you can redeem it as follows:
```

Comment:
Our name in the Docs shouldn't link to the site.  It's already known and linked from a zillion places.